__tests__/kata1.fizzBuzz.test.js

```diff
@@ -1,19 +1,21 @@
-const { fizzBuzz } = require('../src');
+import { fizzBuzz } from '../src/kata1.fizzBuzz';
```

Comment:
As we're working in node, the `import {} from ""`  syntax doesn't work. We need to use the `require` syntax here as we do in all the other tests. This is why our tests for fizzbuzz don't run at all atm with the error `TypeError: (0 , _kata.fizzBuzz) is not a function`